journalbeat/_meta/fields.common.yml

```diff
@@ -317,3 +317,10 @@
       path: event.created
       migration: true
 
+    - name: container
+      type: group
+      fields:
+        - name: tag
```

Comment:
We discussed internally if `container.log.tag` is perhaps the best match for now. The field does not really match with any other fields so we have to put it into a place where it's unlikely to conflict with others in the future. I worry we might introduce `container.tags` one day which would not match well with the current proposal.